jellyfin_kodi/objects/musicvideos.py

```diff
@@ -79,6 +79,12 @@ def musicvideo(self, item, e_item):
                 update = False
                 LOG.info("MvideoId %s missing from kodi. repairing the entry.", obj['MvideoId'])
 
+        if (obj.get('ProductionYear') or 0) > 9999:
```

Comment:
```suggestion
        if (obj.get('ProductionYear', 0) > 9999:
```

wouldn't it be cleaner to use the built in fallback method of `.get()` instead of the `or`?

Reply:
That's what I did first, but obj contains `None` values, so I got an error attempting to `>` `None` and a number.